cloud_formation/configs/cachedb.py

```diff
@@ -366,8 +366,7 @@ def generate(session, domain):
     config.generate()
 
 
-def create(session, domain):
-    """Create the configuration, and launch it"""
+def build_user_data(session, domain):
```

Comment:
This works, normally the user_data is constructed in create_config, so that the user_data object is located next to the resource using it.

Reply:
Yeah, that makes sense!